core/device_hooks/common_kernels.inc.cpp

```diff
@@ -529,11 +533,44 @@ GKO_STUB_VALUE_AND_INDEX_TYPE(GKO_DECLARE_COO_FILL_IN_MATRIX_DATA_KERNEL);
 GKO_STUB_VALUE_AND_INDEX_TYPE(GKO_DECLARE_COO_CONVERT_TO_CSR_KERNEL);
 GKO_STUB_VALUE_AND_INDEX_TYPE(GKO_DECLARE_COO_CONVERT_TO_DENSE_KERNEL);
 GKO_STUB_VALUE_AND_INDEX_TYPE(GKO_DECLARE_COO_EXTRACT_DIAGONAL_KERNEL);
+GKO_STUB_VALUE_AND_INDEX_TYPE(GKO_DECLARE_COO_CONVERT_TO_BCCOO_KERNEL);
+GKO_STUB_VALUE_AND_INDEX_TYPE(GKO_DECLARE_MEM_SIZE_BCCOO_KERNEL);
+/*
+template <typename IndexType>
+GKO_DECLARE_MEM_SIZE_BCCOO_KERNEL(IndexType)
+GKO_NOT_COMPILED(GKO_HOOK_MODULE);
+GKO_INSTANTIATE_FOR_EACH_INDEX_TYPE(GKO_DECLARE_MEM_SIZE_BCCOO_KERNEL);
+*/
 
 
 }  // namespace coo
 
 
+// TODO (script:bccoo): adapt this block as needed
```

Comment:
Also, please remove all `// TODO (script:bccoo)` comments (should be fairly easy to search for all occurrences).

Reply:
I have maintained the TODO comments in the CUDA and HIP files, because the bccoo format are not implemented in these executors yet.